debian/rules

```diff
@@ -69,3 +69,5 @@ override_dh_strip:
 # Upstream encourages the use of 'build' as the developer build output
 # directory, allow that directory to be present and still allow dh to work.
 .PHONY: build
+build:
```

Comment:
The phony line causes the pattern rule at the top to not match 'build' so 'make -f debian/rules build' did nothing. Adding an empty edge causes the pattern rule to match again.

Weird quirk of make.